### DISCUSSION TITLE
Combine en_coreference_web_trf with en_core_web_sm?

### DISCUSSION THREAD

**Discussion by afriedman412** [explosion/spaCy#12598] [Help: Coding & Implementations] [answered]
I'm working on a project that uses coreferencing and more basic tagging/lemma etc functionality. I've been loading two separate language modules (en_coreference_web_trf and en_core_web_sm) and just using each one as needed.

So my questions are:
1. Would it be any faster if I made a version of en_coreference_web_trf with the requisite pipes from en_core_web?
2. Is there an easy (?) way to do that, given that just adding the pipes by name and intializing still requires a bit of work?

**Comment by bdura** [comment, marked as answer]
Hello @afriedman412, you can combine components from two initialized pipelines, although it will not give you any speed-up. There's no _very_ easy way to do that yet, but you can find an example [here](https://github.com/explosion/spaCy/discussions/11585#discussioncomment-4223712).
